SimTKcommon/Simulation/include/SimTKcommon/internal/MeasureImplementation.h

```diff
@@ -1289,6 +1289,43 @@ template <class T>
 class Measure_Differentiate_Result {
 public:
     Measure_Differentiate_Result() : derivIsGood(false) {}
+
+    Xml::Element toXmlElement(const std::string& name) const {
+        static const int version = 1;
+        Xml::Element e("Measure_Differentiate_Result");
+        if (!name.empty()) e.setAttributeValue("name", name);
+        e.setAttributeValue("version", String(version));
+        e.appendNode(toXmlElementHelper(operand, "operand", true));
+        e.appendNode(toXmlElementHelper(operandDot, "operandDot", true));
+        e.appendNode(toXmlElementHelper(derivIsGood, "derivIsGood", true));
```

Comment:
Then I think it would be good to preface the names with `SimTK_` for the measures, and that OpenSim should preface with `OpenSim_`, just to avoid any potential clashes.

Another question: would anything we do here eventually change behavior if [Unified Call Syntax](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2015/n4474.pdf) becomes part of the standard?